src/99-appendices/06-meg-file-formats.md

```diff
@@ -52,21 +52,45 @@ To learn more about CTF’s data organization:
 
 ## Neuromag/Elekta/MEGIN
 
-Neuromag/Elekta/MEGIN data and Tristan Technologies BabyMEG data is stored with
-file extension `.fif`. The digitized positions of the head points are saved
-inside the fif file along with the MEG data, with typically no `_headshape`
-file.
+Neuromag/Elekta/MEGIN and Tristan Technologies BabyMEG data is stored as
+FIFF files with the extension `.fif`. The digitized positions of the head
+points are saved inside the FIFF file along with the MEG data, with typically no
+`_headshape` file.
 
 ```Text
 sub-<label>[_ses-<label>]_task-<label>[_run-<index>]_meg.fif
 ```
 
-Note that we do not provide specifications for cross-talk and
-fine-calibration matrix files in the current version of BIDS.
+### Cross-talk and fine-calibration files
 
-Example:
+In case internal active shielding (IAS) was used during acquisition, raw FIFF
+files need to be processed using Maxwell filtering
+(signal-space separation, SSS) to make the data usable.
+To this end, two specific files are needed:
+The *cross-talk* file, and the *fine-calibration* file,
+both of which are produced by the MaxFilter software
+and the work of the Neuromag/Elekta/MEGIN engineers during maintenance of the MEG acquisition system.
+Both files are thus specific to the site of recording and may change in the
+process of regular system maintenance.
+
+In BIDS, the cross-talk and fine-calibration files are shared unmodified,
+but with BIDS file naming convention and by using the `acq` entity.
+
+-   cross-talk file template: `[sub-<label>][_ses-<label>][_]acq-crosstalk_meg.dat`
+-   fine-calibration file template: `[sub-<label>][_ses-<label>][_]acq-calibration_meg.fif`
```

Comment:
Shouldn't that be
```suggestion
-   cross-talk file template: `[sub-<label>][_ses-<label>]_acq-crosstalk_meg.dat`
-   fine-calibration file template: `[sub-<label>][_ses-<label>]_acq-calibration_meg.fif`
```
?

Reply:
Ah no, never mind! Forgot about inheritance.

Reply:
yep, inheritance is the reason for this form of the template.